extern/ZAPDUtils/Utils/StringHelper.h

```diff
@@ -23,5 +24,8 @@ class StringHelper
 	static int64_t StrToL(const std::string& str, int32_t base = 10);
 	static std::string BoolStr(bool b);
 	static bool HasOnlyDigits(const std::string& str);
+	static bool IsValidHex(std::string_view str);
+	static bool IsValidHex(const std::string& str); 
+	static bool IsValidOffset(std::string_view str);
```

Comment:
is this providing everything needed for https://github.com/HarbourMasters/Shipwright/pull/2292? it's not clear why we'd need both string and string view versions of `IsValidHex` but not of `IsValidOffset`, if anything i'd think it'd be the opposite